docs/csharp/language-reference/statements/jump-statements.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Jump statements - C# reference"
+description: "Learn about C# jump statements: break, continue, return, and goto."
+ms.date: 12/06/2021
+f1_keywords:
+  - "break_CSharpKeyword"
+  - "continue_CSharpKeyword"
+  - "return_CSharpKeyword"
+  - "goto_CSharpKeyword"
+helpviewer_keywords:
+  - "break statement [C#]"
+  - "break keyword [C#]"
+  - "continue statement [C#]"
+  - "continue keyword [C#]"
+  - "return statement [C#]"
+  - "return keyword [C#]"
+  - "goto statement [C#]"
+  - "goto keyword [C#]"
+---
+# Jump statements (C# reference)
+
+The following statements unconditionally transfer control:
+
+- The [`break` statement](#the-break-statement): terminates the closest enclosing [iteration statement](iteration-statements.md) or [`switch` statement](selection-statements.md#the-switch-statement).
+- The [`continue` statement](#the-continue-statement): starts a new iteration of the closest enclosing [iteration statement](iteration-statements.md).
+- The [`return` statement](#the-return-statement): terminates execution of the function in which it appears and returns control to the caller.
+- The [`goto` statement](#the-goto-statement): transfers control to a statement that is marked by a label.
+
+## The `break` statement
+
+The `break` statement terminates the closest enclosing [iteration statement](iteration-statements.md) (that is, `for`, `foreach`, `while`, or `do` loop) or [`switch` statement](selection-statements.md#the-switch-statement). The `break` statement transfers control to the statement that follows the terminated statement, if any.
+
+:::code language="csharp" interactive="try-dotnet-method" source="snippets/jump-statements/BreakStatement.cs" id="BasicExample":::
+
+In nested loops, the `break` statement terminates only the innermost loop that contains it, as the following example shows:
+
+:::code language="csharp" interactive="try-dotnet-method" source="snippets/jump-statements/BreakStatement.cs" id="NestedLoop":::
+
+When you use the `switch` statement inside a loop, a `break` statement at the end of a switch section transfers control only out of the `switch` statement. The loop that contains the `switch` statement is unaffected, as the following example shows:
+
+:::code language="csharp" source="snippets/jump-statements/BreakStatement.cs" id="SwitchInsideLoop":::
+
+## The `continue` statement
+
+The `continue` statement starts a new iteration of the closest enclosing [iteration statement](iteration-statements.md) (that is, `for`, `foreach`, `while`, or `do` loop), as the following example shows:
+
+:::code language="csharp" interactive="try-dotnet-method" source="snippets/jump-statements/ContinueStatement.cs" id="BasicExample":::
+
+## The `return` statement
+
+The `return` statement terminates execution of the function in which it appears and returns control and the function's result, if any, to the caller. The `return` statement can be any of the following two forms:
```

Comment:
Regarding the next question, I might consider something like this:

```suggestion
The `return` statement terminates execution of the function in which it appears and returns control and the function's result, if any, to the caller. The `return` statement has an expression following the `return` keyword when the member produce a value. The `return` statement must not have an expression when the member doesn't prodcue a value. When the `return` statement has an expression, that expression must be convertible to the type returned by the member. The only exception to this rule is `async` functions, where the expression must be convertible to the type argument for <xref:System.Threading.Tasks.Task%601?displayProperty=nameWithType>. Return statements in `async` methods that return <xref:System.Threading.Tasks.Task?displayProperty=nameWithType> must not have an expression.
```

Reply:
@BillWagner thank you, I've rewritten this section based on your suggestion. Please review it again.